ca/certificate-authority.go

```diff
@@ -267,7 +267,12 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 		return emptyCert, err
 	}
 
-	notAfter := time.Now().Add(ca.ValidityPeriod)
+	validity := ca.ValidityPeriod
+	if validityDays > 0 {
```

Comment:
We should also have a minimum number of validityDays. Clients issuing extremely short-lived certs (1 day) will cause unnecessary load and possibly raise some BR / CPS issues.